package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "open": "open http://localhost:6060",
+    "c": "yarn run plop component",
```

Comment:
@cisacke The `yarn run c` is just a shorthand. The full command is the description.